src/downshift.js

```diff
@@ -67,7 +68,12 @@ class Downshift extends Component {
     defaultIsOpen: false,
     getA11yStatusMessage,
     id: generateId('downshift'),
-    itemToString: i => (i == null ? '' : String(i)),
+    itemToString: i => {
+      if (isPlainObject(i)) {
```

Comment:
I'd prefer this to only happen in dev. I think the build is wired up to handle `process.env.NODE_ENV` properly. Could you make this: `if (process.env.NODE_ENV !== 'production' && isPlainObject(i)) {`

Reply:
Would `process.env.NODE_ENV === 'development'` more prefereable in this instance?